src/source/PeerConnection/SessionDescription.c

```diff
@@ -230,7 +230,7 @@ STATUS setPayloadTypesFromOffer(PHashTable codecTable, PSessionDescription pSess
     return retStatus;
 }
 
-STATUS setTransceiverPayloadTypes(PHashTable codecTable, PDoubleList pTransceievers) {
+STATUS setTransceiverPayloadTypes(PHashTable codecTable, PDoubleList pTransceiver) {
```

Comment:
`pTransceievers` I think is the right name since it is a list of them

Reply:
+1

Reply:
hmm, one typo to another typo, will fix it.